documentation/release_notes.rst

```diff
@@ -8,6 +8,22 @@ The Intel® oneAPI DPC++ Library (oneDPL) accompanies the Intel® oneAPI DPC++/C
 and provides high-productivity APIs aimed to minimize programming efforts of C++ developers
 creating efficient heterogeneous applications.
 
+New in 2022.0.0
+===============
+
+New Features
+------------
+- Added ``<complex>`` header functionality as `Tested Standard C++ API`_.
+- Improved performance of ``sort`` and ``stable_sort``  algorithms on GPU devices when using Radix sort [#fnote1]_.
+
+Fixed Issues
+------------
+- Fixed error in ``oneapi::dpl::experimental::ranges::guard_view`` and ``oneapi::dpl::experimental::ranges::zip_view`` when using  ``operator[]`` with a very large index.
+
```

Comment:
Does it make sense to somehow mention "[oneDPL] Added support C++ functor/lambda  for permutation_iterator  (#603)" done by Mikhail?

Reply:
Please, see my comment above.

Reply:
Added the suggestion